<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewStiker</class>
 <widget class="QWidget" name="CreateNewStiker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTextEdit" name="te_text"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Color stiker</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rb_yellow">
          <property name="text">
           <string>Yellow</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_lime">
          <property name="text">
           <string>Lime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_pink">
          <property name="text">
           <string>Pink</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_orange">
          <property name="text">
           <string>Orange</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_blue">
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_create">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
